app/services/twilio_service.rb

```diff
@@ -77,4 +92,8 @@ def sanitize_errors
   def sanitize_phone_number(str)
     str.gsub!(/\+[\d\(\)\- ]+/) { |match| match.gsub(/\d/, '#') }
   end
+
+  def format_phone_number(str)
+    str.gsub!(/\s|\(|\)|\-/,'')
```

Comment:
I would prefer that we do `gsub` without the `!` so we don't mutate the caller. The reason for mutating the string above was to sanitize an exception where we definitely do want to mutate